packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -761,6 +761,38 @@ const TESTS = new Map([
       { test: 'notInclude', expected: 'utm_source=email' },
     ]],
   ])],
+  ['verifyCodeEmail', new Map([
+    ['subject', { test: 'equal', expected: `Verification code: ${MESSAGE.code}` }],
+    ['headers', new Map([
+      ['X-SES-MESSAGE-TAGS', { test: 'equal', expected: sesMessageTagsHeaderValue('verifyEmail') }],
+      ['X-Template-Name', { test: 'equal', expected: 'verifyCodeEmail' }],
+      ['X-Template-Version', { test: 'equal', expected: TEMPLATE_VERSIONS.verifyCodeEmail }],
+      ['X-Verify-Short-Code', { test: 'equal', expected: MESSAGE.code }],
+    ])],
+    ['html', [
+      { test: 'include', expected: configHref('privacyUrl', 'welcome', 'privacy') },
+      { test: 'include', expected: configHref('supportUrl', 'welcome', 'support') },
+      { test: 'include', expected: 'Is this you signing up?' },
+      { test: 'include', expected: `IP address: ${MESSAGE.ip}` },
+      { test: 'include', expected: `${MESSAGE.location.city}, ${MESSAGE.location.stateCode}, ${MESSAGE.location.country} (estimated)` },
+      { test: 'include', expected: `${MESSAGE.uaBrowser} on ${MESSAGE.uaOS} ${MESSAGE.uaOSVersion}` },
+      { test: 'include', expected: 'If yes, use this verification code:' },
+      { test: 'include', expected: MESSAGE.code },
+      { test: 'include', expected: "It expires in 20 minutes." },
+      { test: 'notInclude', expected: 'utm_source=email' },
+    ]],
+    ['text', [
+      { test: 'include', expected: `Mozilla Privacy Policy ${configUrl('privacyUrl', 'welcome', 'privacy')}` },
+      { test: 'include', expected: `For more information, please visit ${configUrl('supportUrl', 'welcome', 'support')}` },
+      { test: 'include', expected: 'Is this you signing up?' },
+      { test: 'include', expected: `IP address: ${MESSAGE.ip}` },
+      { test: 'include', expected: `${MESSAGE.location.city}, ${MESSAGE.location.stateCode}, ${MESSAGE.location.country} (estimated)` },
+      { test: 'include', expected: 'If yes, use this verification code:' },
+      { test: 'include', expected: MESSAGE.code },
```

Comment:
...whereas these two I think would be improved by combining them so that we're asserting the correct position of `MESSAGE.code`:

```js
{ test: 'include', expected: `If yes, use this verification code:\n${MESSAGE.code}` },
```

(doesn't work for the `html` assertions sadly, but just doing it here gives us some extra coverage)